api/v1beta2/ibmvpcmachine_types.go

```diff
@@ -36,10 +39,23 @@ type IBMVPCMachineSpec struct {
 	// Name of the instance.
 	Name string `json:"name,omitempty"`
 
+	// CatalogOffering is the Catalog Offering OS image which would be installed on the instance.
+	// An OfferingCRN or VersionCRN is required, the PlanCRN is optional.
+	// +optional
+	CatalogOffering *IBMCloudCatalogOffering `json:"catalogOffering,omitempty"`
```

Comment:
For my understanding, how would this respective image would be imported to VPC or the global catalog images are generally available to all the VPC?

Reply:
Alos since this catalogOffering is specific to os image should we rename the member  to indicate that?

Reply:
Sure, I can rename it.
In the cases I have used, this documentation provides details on creating a Private Catalog:  https://cloud.ibm.com/docs/account?topic=account-restrict-by-user&interface=ui
And details on creating a sharable VPC Custom Image:  https://cloud.ibm.com/docs/vpc?topic=vpc-custom-image-cloud-private-catalog&interface=ui